log-visualizer/common.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
```

Comment:
This code previously sits in `visualizer.ipynb`. I moved it out so that we can reuse it in multiple notebooks.